index.bs

```diff
@@ -1477,7 +1486,9 @@ The [=authenticator data=] structure is a byte array of 37 bytes or more, as fol
         <td>
             Flags (bit 0 is the least significant bit):
             - Bit 0: [=Test of User Presence=] (`TUP`) result.
-            - Bits 1-5: Reserved for future use (`RFU`).
+            - Bit 1: Reserved for future use (`RFU1`).
```

Comment:
How does this bit affect the client's and/or RP's behaviors? What "if" statements do they write based on this condition? Is there valuable interoperable behavior we should be specifying based on this result?